doc/releases/changelog-dev.md

```diff
@@ -135,6 +135,9 @@
 * A function called `apply_operation` has been added to the new `qutrit_mixed` module found in `qml.devices` that applies operations to device-compatible states.
   [(#5032)](https://github.com/PennyLaneAI/pennylane/pull/5032)
 
+* A function called `measure` has been added to the new `qutrit_mixed` module found in `qml.devices` that measures device-compatible states for a collection of measurement processes.
```

Comment:
It would be more concise to merge all the relevant device additions into a single CHANGELOG entry with all the PRs listed after. Or is that against convention? (Gabe no need to update yet, we can see if anyone has any feedback)

Reply:
Sometimes we do so, see:

https://github.com/PennyLaneAI/pennylane/blob/d0c435d15ddcd9a28f7ff7104fceef6e4304ab42/doc/releases/changelog-dev.md?plain=1#L123

But we also clean up and reorganize the changelog before each release, at which point each point might get combined.

Reply:
Great, let's leave it for now then!